tests/cron_job_tests/test_genomic_pipeline.py

```diff
@@ -169,6 +188,7 @@ def _write_cloud_csv(file_name, contents_str, bucket=None, folder=None):
         n = clock.CLOCK.now()
         ntime = time.mktime(n.timetuple())
         os.utime(provider.get_local_path(path), (ntime, ntime))
+        return cloud_file
```

Comment:
Where is this return used? And does this work? I'm not sure of how the scoping works, but I thought the file wouldn't be usable outside of the scope of the context manager that opens it.

Reply:
Yea it does work. Just used for getting the local RAW path of the CSV which is written to something like `var/tmp` , for verifying the physical columns/row(s) of the local CSV that is outputted.

<img width="902" alt="Screen Shot 2021-07-27 at 12 39 32 PM" src="https://user-images.githubusercontent.com/5114837/127193577-ab63549c-14bd-4a1c-870f-8a665b6be1d8.png">